zerver/lib/export.py

```diff
@@ -279,6 +279,7 @@
 # TODO: This data structure could likely eventually be replaced by
 # inspecting the corresponding Django models
 DATE_FIELDS: Dict[TableName, List[Field]] = {
+    'zerver_analytics': ['date_created'],
```

Comment:
@arpit551 do you understand what the `zerver_analytics` key is here?  I don't understand it -- it's not a real database table.

Reply:
@timabbott I think the Table name is incorrect here. While exporting analytics tables we use `normal_parent=analytics_config`
and the analytics_config is basically same as realm_config. We use the seed_object `realm` here so that we can get the analytics data of that particular realm. So in the final analytics.json, there is duplicate data of `zerver_realm` in the name of `zerver_analytics`.  This was introduced in [https://github.com/zulip/zulip/pull/11402/commits/226fc3adc88c2c04eb975486bfc031c632d3f110](https://github.com/zulip/zulip/pull/11402/commits/226fc3adc88c2c04eb975486bfc031c632d3f110)

Reply:
I'll submit a PR once this merges.

Reply:
Just make a PR stacked on these changes; that should work fine.